src/StringMisc.java

```diff
@@ -0,0 +1,57 @@
+public class StringMisc {
+
+        public static String norm(String name) {
+            // Check special case if name input is empty
+            if (name == null) {
+                return "";
+            }
+            return name.substring(0, 1).toUpperCase() + name.substring(1).toLowerCase();
+        }
+
+        public static String init(String name) {
+            String[] nameParts = name.split(" ");
+            StringBuilder result = new StringBuilder();
+
+            // Process all but the last name
+            for (int i = 0; i < nameParts.length - 1; i++) {
+                result.append(nameParts[i].substring(0, 1).toUpperCase()).append(". ");
+            }
+
+            // Process the last name
+            result.append(nameParts[nameParts.length - 1].substring(0, 1).toUpperCase()).append(nameParts[nameParts.length - 1].substring(1).toLowerCase());
```

Comment:
move the second append to a new line. In general, let's keep the line length under 80 characters to make it more readable